app/src/main/java/com/github/se/cyrcle/ui/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,192 @@
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import coil.request.ImageRequest
+import coil.transform.CircleCropTransformation
+import com.github.se.cyrcle.model.user.User
+import com.github.se.cyrcle.ui.theme.ColorLevel
+import com.github.se.cyrcle.ui.theme.getButtonColors
+
+@Composable
+fun ProfileScreen(user: User) {
+  var isEditing by remember { mutableStateOf(false) }
+  var firstName by remember { mutableStateOf(TextFieldValue(user.firstName)) }
+  var lastName by remember { mutableStateOf(TextFieldValue(user.lastName)) }
+  var username by remember { mutableStateOf(TextFieldValue(user.username)) }
+
+  var favoriteParkings by remember { mutableStateOf(user.favoriteParkings.toMutableList()) }
+  val parkingFavorites = remember {
+    mutableStateListOf(*List(user.favoriteParkings.size) { true }.toTypedArray())
+  }
+
+  var profilePictureUrl by remember { mutableStateOf(user.profilePictureUrl) }
+  val defaultImageUrl =
```

Comment:
Instead of fetching from online every time this screen is composed, add as a ressource to the project

Reply:
You could use a material 3 icon for this purpose too, giving us more liberty in shapes, colors, etc.